### ISSUE TITLE
bug + fix for NullPointerException when using EMFConverter or EMF2SVG

### ISSUE THREAD

**Issue by JoDa-Itec** [freehep/freehep-vectorgraphics#30]
An exception is raised by EMFConverter.java:81

This can simply be solved by:
- add "import java.util.Properties;" to line 7
- move "Properties p = new Properties();" at line 69 one line up, out of the comment
- change "null" into "p" at line 85

**Comment by chewi** [comment]
Good call. I have [patched this in Gentoo](https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-java/freehep-graphicsio-emf/files/github-30.patch).
